addons/point_of_sale/static/src/app/hooks/use_tours.js

```diff
@@ -0,0 +1,63 @@
+import { tourState } from "@web_tour/tour_service/tour_state";
+import { TourSelectorPopup } from "../components/tour_selector_popup/tour_selector_popup";
+import { makeAwaitable } from "../store/make_awaitable_dialog";
+import { useService } from "@web/core/utils/hooks";
+import { useState } from "@odoo/owl";
+
+export default function useTours() {
+    const tour = useService("tour_service");
+    const dialog = useService("dialog");
+    const states = useState({
+        selectedTours: new Set(),
+        running: false,
+        index: 0,
+    });
```

Comment:
```suggestion
    const states = {
        selectedTours: new Set(),
        running: false,
        index: 0,
    };
```

doesn't need to be a state. Nothing seems to be subscribing (using) to it.